docs/circuits.ipynb

```diff
@@ -41,14 +41,28 @@
         "\n"
       ]
     },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "g_TubHnpIyUH",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "!pip install --quiet cirq\n",
+        "import cirq"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
     {
       "cell_type": "markdown",
       "metadata": {
         "id": "77zB_UqkAwCR",
         "colab_type": "text"
       },
       "source": [
-        "![Circuits and Moments](https://cirq.readthedocs.io/en/latest/_static/CircuitMomentOperation.png)"
+        "![Circuits and Moments](https://cirq.readthedocs.io/en/stable/_images/CircuitMomentOperation.png)"
```

Comment:
Is this image not in `latest`? I thought we store all these things in `static`. Why the change in this link?

Reply:
Good catch.  I think this was a merge issue of some sorts, since I changed this in an early PR.
I will also go and double-check other changes to make sure I am not squashing anything once I update the branch.